frontend/src/components/explorer/gemBrowser/Metabolite.vue

```diff
@@ -103,12 +115,12 @@ export default {
     MapsAvailable,
     ReactionTable,
     GemContact,
+    GemSearch,
   },
   data() {
     return {
-      messages,
       type: 'metabolite',
-      metaboliteId: '',
+      metaboliteId: ' ',
```

Comment:
Why is a space added here?

Reply:
Because it was used before it was set, and Vue was complaining about the empty string. This should be revisited.